AppOne/AppOne/HomeVC.swift

```diff
@@ -0,0 +1,16 @@
+//
+//  HomeVC.swift
+//  AppOne
+//
+//  Created by Asapu, Ajaikumar on 19.11.22.
+//
+
+import UIKit
+
+final class HomeVC: UIViewController {
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        view.backgroundColor = .green
```

Comment:
for light and darkmode support we should use the .system colors instead of this.

Reply:
I know this is a placeholder thing, but for the future to remember. 💯

Reply:
Updated [here](https://github.com/JoeHoHH/ajayTheLead/pull/1/commits/a4e107c44a0e219e09b5d3c6165c235a278758de)